api-report/odsp-driver.api.md

```diff
@@ -63,7 +63,7 @@ export function getApiRoot(origin: string): string;
 export function getHashedDocumentId(driveId: string, itemId: string): Promise<string>;
 
 // @public
-export function getLocatorFromOdspUrl(url: URL): OdspFluidDataStoreLocator | undefined;
+export function getLocatorFromOdspUrl(url: URL, requireFluidSignature?: boolean): OdspFluidDataStoreLocator | undefined;
```

Comment:
Documentation seem to indicate that parameter will be falsy when omitted, in fact, it will be truthy

Reply:
This is the file generated by api-extractor. Is there a proper way to ensure that this documentation is up-to-date with the changes in this PR? I'm not seeing any changes when I run `pnpm build:fast odsp-driver` or `pnpm build:docs`

Reply:
The PR build will fail if the docs are not correct - we run a check to ensure the build doesn't generate git-wise changes.

Reply:
Sounds like the documentation here is ok then, resolving conversation

Reply:
Hmm, documentation is factually incorrect. I wonder if this is a bug with api-extractor.